### PR TITLE
Expand "Installation Setup" section

### LINKED ISSUE #19: Write "Installation Setup" handbook section.
> https://athensresearch.gitbook.io/handbook/community/athens-guide/getting-started/installation-guide

> https://github.com/athensresearch/handbook/blob/main/community/athens-guide/getting-started/installation-guide.md

### PR DESCRIPTION
**Pull request by elazar**
Fixes #19

### CODE REVIEW

File: community/athens-guide/getting-started/installation-guide.md [hunk 1]
```diff
@@ -1,22 +1,101 @@
...
-* Run the [Athens backend](https://github.com/athensresearch/athens-backend) on a system, like a virtual private server \(VPS\) or [Heroku deployment](https://github.com/athensresearch/athens-backend#heroku) that you can access from every machine or network.
+1. Use a file synchronization tool like [Dropbox](https://www.dropbox.com) or [OneDrive](https://www.microsoft.com/en-us/microsoft-365/onedrive/online-cloud-storage). While Athens warns you about updated blocks coming from these tools, you should take care not to edit the same database from multiple machines at the same time.
+2. Run the [Athens backend](https://github.com/athensresearch/athens-backend) on a system like a [virtual private server](https://en.wikipedia.org/wiki/Virtual_private_server) (VPS) or [Heroku deployment](https://github.com/athensresearch/athens-backend#heroku) that you can access from every machine or network. This backend is in the process of being refactored into the [main Athens git repository](https://github.com/athensresearch/athens) (see [this issue](https://github.com/athensresearch/athens/pull/1170/) for details), but until that effort is completed, the existing archived backend can be used.
```
Comment (joelhans) [inline community/athens-guide/getting-started/installation-guide.md:95]: Let's remove the first sentence about the backend. We're actually not recommending that users install the backend any more, as the implementation isn't the best experience.

File: community/athens-guide/getting-started/installation-guide.md [hunk 2]
```diff
@@ -1,22 +1,101 @@
...
+* [Installing Athens](#installing-athens)
+  * [Windows](#windows)
+  * [OS X](#os-x)
+  * [Linux](#linux)
+* [Post-installation](#post-installation)
+  * [Finding where your data is stored](#finding-where-your-data-is-stored)
+  * [Importing an existing database](#importing-an-existing-database)
+  * [Creating a new database](#creating-a-new-database)
+  * [Syncing your data between machines](#syncing-your-data-between-machines)
+  * [Updating Athens](#updating-athens)
```
[the remark this answers is not in the document]
Comment (elazar) [reply community/athens-guide/getting-started/installation-guide.md:12]: By "this" you mean... the table of contents?

[the remark this answers is not in the document]
Comment (joelhans) [reply community/athens-guide/getting-started/installation-guide.md]: Yes, the table of contents. There already is one on the right-hand side on GitBook, so I'm just wondering about the value in having this at the top vs. getting straight into the content.

[the remark this answers is not in the document]
Comment (elazar) [reply community/athens-guide/getting-started/installation-guide.md]: Fair enough. I removed it.

**Review by joelhans — APPROVED** [review]
Approved—thanks very much. You're definitely doing it right. I'll be faster on any future reviews now that I'm back home and in the normal flow.

**Pull request merged**